doc/examples/collision_detection/lib/main.dart

```diff
@@ -194,7 +192,7 @@ class MyGame extends BaseGame with HasCollidables, HasDraggableComponents {
     MyCollidable lastToAdd = snowman;
     add(screen);
     add(snowman);
-    int totalAdded = 1;
+    var totalAdded = 1;
```

Comment:
This may be only my opinion, but I rather prefer the old style. Var just seems wrong to me hahaha, but this may be only me.

Reply:
var is the same as final that we already use. the rule I propose is: local variables are never explicitly typed. fields/methods are always explicitly typed.

in this case int is quite short but most of the time the type declaration would be pretty big and uncessary.

Reply:
I know, but I think I am too hurt from JS so `var` always looks weird to me. But yeah, I like that rule

Reply:
I had exactly the same feeling as you Erick! But I think this will be fine, and it follows the dart styleguide.